sorc/ncep_post.fd/MDL2AGL.f

```diff
@@ -61,6 +62,7 @@ SUBROUTINE MDL2AGL
                             jsta_2l, jend_2u, im, jm, jsta, jend, imp_physics
       use rqstfld_mod,  only: iget, lvls, iavblfld, lvlsxml, id
       use gridspec_mod, only: gridtype
+      use UPP_PHYSICS
```

Comment:
Is new module needed in this subroutine?

Reply:
CALRH was mentioned in the doc block. Though it's not called in the code, UPP_PHYSICS is used for place holder if developers want to add CALRH in the future. It can be commented out.